packages/style-loader/index.js

```diff
@@ -52,23 +55,40 @@ module.exports = (neutrino, opts = {}) => {
   }
 
   rules.forEach(options => {
-    neutrino.config.module
-      .rule(options.ruleId)
+    const styleRule = neutrino.config.module.rule(options.ruleId);
+    const loaders = [{
+      loader: require.resolve('style-loader'),
+      options: options.style,
+      useId: options.styleUseId
+    },
+    {
+      loader: require.resolve('css-loader'),
+      options: Object.assign(options.css, {
+        importLoaders: options.css.importLoaders + options.loaders.length
+      }),
+      useId: options.cssUseId
+    }, ...options.loaders]
+    .map((loader, index) => {
+      const obj = typeof loader === 'object' ? loader : { loader };
+
+      return Object.assign(obj, {
+        useId: obj.useId || `${options.cssUseId}-${index}`
```

Comment:
I like this, but this will need to be clearly noted in the documentation, that without the `useId`, it will generate an id in this format.

Reply:
I already have the docs writte in #574.